src/FilterResource.php

```diff
@@ -129,6 +129,19 @@ public function lower()
     }
 
     /**
+     * Returns rule that formats numbers
+     *
+     * @param int $decimals
+     * @param string $decimalPoint
+     * @param string $thousandSeperator
+     * @return FilterResource
```

Comment:
Can you change this to `@return $this` so that we are sure the editor will understand the auto-completion in case of an extended filter.